proptest-derive/src/error.rs

```diff
@@ -229,7 +225,7 @@ macro_rules! fatal {
     ($error: ident ($($arg: ident: $arg_ty: ty),*), $code: ident,
      $msg: expr, $($fmt: tt)+) => {
         pub fn $error<T>(ctx: Ctx, $($arg: $arg_ty),*) -> DeriveResult<T> {
-            ctx.fatal(format!(mk_err_msg!($code, $msg), $($fmt)+))
+            ctx.fatal(mk_err_msg!($code, format!($msg, $($fmt)+)))
```

Comment:
I’m making sure only `$msg` is used as the format string of `format!` because `to_lowercase` isn’t `const`.